application/comit_node/src/swap_protocols/mod.rs

```diff
@@ -15,8 +16,27 @@ pub use self::{
     timestamp::Timestamp,
 };
 
-#[derive(Debug)]
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq, Eq, Hash, Copy)]
+pub enum HashFunction {
+    #[serde(rename = "sha-256")]
+    SHA256,
```

Comment:
:upside_down_face: Rust naming conventions [recommend](https://doc.rust-lang.org/1.0.0/style/style/naming/README.html) treating acronyms as one word e.g. `Rfc003`
```suggestion
    Sha256,
```

Reply:
Cool, please continue to always tell me these things - it is super useful for me learning Rust.  Will fix.